plugin/src/main/java/io/deephaven/plugin/js/Paths.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.plugin.js;
+
+import java.nio.file.Path;
+
+public interface Paths {
```

Comment:
I'm not sure how to best rename it if necessary - I'm personally less offended by names that overlap in these regards, although I know others disagree. At least `java.nio.file.Paths` is _somewhat_ discouraged in their javadoc:

```java
 * @apiNote
 * It is recommended to obtain a {@code Path} via the {@code Path.of} methods
 * instead of via the {@code get} methods defined in this class as this class
 * may be deprecated in a future release.
```